addon/components/combobox/-label.hbs

```diff
@@ -0,0 +1,11 @@
+{{#let (element (or @as 'label')) as |Tag|}}
+  <Tag
+    id='{{@guid}}-label'
+    for='{{@guid}}-button'
+    {{on 'click' @handleLabelClick}}
+    {{did-insert @registerLabelElement}}
```

Comment:
Planning to migrate this to `ember-modifier` in a future PR (needs to be done together with its super class Listbox component)